include/mbedtls/ssl.h

```diff
@@ -107,6 +107,7 @@
 #define MBEDTLS_ERR_SSL_TIMEOUT                           -0x6800  /**< The operation timed out. */
 #define MBEDTLS_ERR_SSL_CLIENT_RECONNECT                  -0x6780  /**< The client initiated a reconnect from the same port. */
 #define MBEDTLS_ERR_SSL_UNEXPECTED_RECORD                 -0x6700  /**< Record header looks valid but is not expected. */
+#define MBEDTLS_ERR_SSL_IGNORE_NON_FATAL                  -0x6680  /**< The message is a non-fatal error. */
```

Comment:
I understand that we might want to have non-fatal error codes, to me these are either warnings or errors that the machine can recover from. However, I would prefer if you were more explicit about what the actual cause of the non-fatal problem is. This seems a bit too generic.